_po/ja/how-to/laravel.po

```diff
@@ -736,6 +1122,14 @@ msgid ""
 "    {\n"
 "        // \\App\\Models\\User::factory(10)->create();"
 msgstr ""
+"class DatabaseSeeder extends Seeder\n"
+"{\n"
+"    /**\n"
+"     * Seed the application's database.\n"
+"     */\n"
+"    public function run(): void\n"
+"    {\n"
+"        // \\App\\Models\\User::factory(10)->create();"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.